### DISCUSSION TITLE
plotting stopped at 94%

### DISCUSSION THREAD

**Discussion by chianew** [Chia-Network/chia-blockchain#3048] [Q&A]
![chia1](https://user-images.githubusercontent.com/83288663/116486833-1cc8ac80-a8c1-11eb-9921-1af3a4348cae.jpg)
![chia2](https://user-images.githubusercontent.com/83288663/116486841-205c3380-a8c1-11eb-8708-1db524772b48.jpg)

This is my second time to get this problem. I had been waiting more than 4 hours but nothing changed. 
Please help me fix it, thank you!

**Comment by myth2loki** [comment]
me too。recent days,， ploting stuck at everywhere。I got stuck 31% 45% 69% 90% and so on....

**Reply by tonywonghs** [reply]
> me too。recent days,， ploting stuck at everywhere。I got stuck 31% 45% 69% 90% and so on....

Same to mine, single plot stuck, parallel plot stuck, but sometimes it was fine. Totally no idea, I first thought my computer issue...

**Comment by sotnikop** [comment]
What's everyone concurrency at and what ssd's are you using to plot? I find its more liable to get stuck at above 6 plots per 2tb MP600

**Reply by chianew** [reply]
WD SN550 1TB and 2TB SSD

**Comment by CCarroz** [comment]
I had a few plots getting stuck and what it turned out to be was temp files stayed in the plot folder and storage was full and thus plotting stopped. Not sure if that helps you out, but I see 1 plots going if you are on a 1TB SSD, chances are you are full.  I see about 260GB needed per plot, and if you have 4 plots going on the same drive, chances are you are full.

**Reply by chianew** [reply]
3 K32 plotting in 1T SSD and 5 K32 plotting in 2T SSD.

**Comment by huiui** [comment]
What is your operating system? I have windows and old mac running fine, and two newer macs keeps getting stuck at 94%..

**Reply by chianew** [reply]
windows 10 professional
